graphrag/config/init_content.py

```diff
@@ -0,0 +1,141 @@
+# Copyright (c) 2024 Microsoft Corporation.
+# Licensed under the MIT License
+
+"""Content for the init CLI command to generate a default configuration."""
+
+import graphrag.config.defaults as defs
+
+INIT_YAML = f"""\
+### This config file contains required core defaults that must be set, along with a handful of common optional settings.
+### For a full list of available settings, see https://microsoft.github.io/graphrag/config/yaml/
+
+### LLM settings ###
+## There are a number of settings to tune the threading and token limits for LLM calls - check the docs.
+
+encoding_model: cl100k_base # this needs to be matched to your model!
+
+llm:
+  api_key: ${{GRAPHRAG_API_KEY}} # set this in the generated .env file
+  type: {defs.LLM_TYPE.value} # or azure_openai_chat
+  model: {defs.LLM_MODEL}
+  model_supports_json: true # recommended if this is available for your model.
+  # audience: "https://cognitiveservices.azure.com/.default"
+  # api_base: https://<instance>.openai.azure.com
+  # api_version: 2024-02-15-preview
+  # organization: <organization_id>
+  # deployment_name: <azure_model_deployment_name>
+
+parallelization:
+  stagger: {defs.PARALLELIZATION_STAGGER}
+  # num_threads: {defs.PARALLELIZATION_NUM_THREADS}
+
+async_mode: {defs.ASYNC_MODE.value} # or asyncio
+
+embeddings:
+  async_mode: {defs.ASYNC_MODE.value} # or asyncio
+  vector_store:{defs.VECTOR_STORE}
```

Comment:
Do we document somewhere how to set up for AAIS?

Reply:
It is on the yaml page now

Reply:
The bullets don't nest well though, so it isn't clear. Checking...